main/document/edit_document.php

```diff
@@ -28,6 +28,7 @@
  */
 
 require_once __DIR__.'/../inc/global.inc.php';
+require_once '../inc/lib/urlUtils.lib.php';
```

Comment:
Please always use __DIR__ as a prefix to allow for later changes that won't be affecting this require.